### ISSUE TITLE
Score is not being updated properly

### ISSUE THREAD

**Issue by justweb1** [Flatracer/DemolitionDerbyGamemode#11]
Score is not being updated in the top right display and the leaderboards only get updated sometimes and not for every player.

**Comment by Flatracer** [comment]
We are aware of the issues with the scoreboard.

The Leaderboard got updated all the time while testing.
If a player dies, it should update the leaderboard.

**Comment by justweb1** [comment]
We must of played 30 rounds on this map. This is the contents of the leaderboard for it.

Default Map 1
```json
{
    "d12dff2dfbb761284cb151697cc32c094739efcd":{"Name":"UnauthorizedGamer","Won":0,"Lost":1},
    "095faadaeb8d1f2c54669648aa367cf3564fc874":{"Name":"Brandon R. 103","Won":0,"Lost":1},
    "fcb38c82ef5ed53ed0541e87bd75429993011ac2":{"Name":"JustWeb","Won":1,"Lost":0}
}
```

**Comment by Flatracer** [comment]
Are there any errors in the Remote Console or in the Client Console of the missing players?

**Comment by justweb1** [comment]
I'll have to test that later. I don't have anyone on right now. I couldn't seem to get the dev menu open.

**Comment by Flatracer** [comment]
You have to add yourself as an admin in the server .cfg file.

**Comment by justweb1** [comment]
I am listed as an admin for the FiveM Server. It looks like F10 should open it from the code. Am I missing something?

**Comment by Flatracer** [comment]
Thats correct. You need [NativeUI by Frazzle](https://github.com/FrazzIe/NativeUILua) started for it to work

**Comment by Flatracer** [comment]
Did you added these: https://github.com/Flatracer/DemolitionDerbyGamemode/wiki/Requirements

**Comment by Flatracer** [comment]
About the leaderboard problem: I might have found the bug. Will test this out and push it to github later.
